probes/entries.go

```diff
@@ -91,6 +92,9 @@ var (
 		hasLicenseFile.Run,
 		hasFSFOrOSIApprovedLicense.Run,
 	}
+	PermissiveLicense = []ProbeImpl{
+		hasPermissiveLicense.Run,
+	}
```

Comment:
if this were added to `Uncategorized`, the probe could be called by those interested. even if the check wasn't added

Reply:
Done